ethereum/test/pyth.js

```diff
@@ -1115,9 +1105,15 @@ contract("Pyth", function () {
       2
     );
 
-    await expectRevert(
-      this.pythProxy.executeGovernanceInstruction(transferBackVaaWrong),
-      "cannot upgrade to an older governance data source"
+    // Strangely this test was failing without the hard coded gas limit.
+    // It will failing randomly on a place that it shouldn't without any message due to
+    // gas limit if the value is not hard-coded.
+    // Gas value 6.7m gas is very high and close to the ganache limit.
```

Comment:
nitpick: comments like this should be written in the present tense:

"this test fails without a hard coded gas limit."

Reply:
yeah you are right. thanks.